Automattic/jetpack
# Issue Templates: update bug report template

## Proposed changes:

We're updating our priority matrix, and consequently need to update the information we collect in our issues, so our GitHub action can automatically assign priority.

This also consolidates the steps to reproduce into one field instead of 3, to make things simpler for reporters.

### Other information:


## Jetpack product discussion

- Related PR: https://github.com/Automattic/wp-calypso/pull/97049/
- pfVjQF-su-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

Open the bug issue template here on GitHub and check the preview ; the impact assessment section should match the one introduced in Calypso, in https://github.com/Automattic/wp-calypso/pull/97049/
